docker/Dockerfile

```diff
@@ -1,14 +1,22 @@
 FROM alpine:3.5
 
+# Enables customized options using environment variables
 ENV BACKEND='http://localhost:5000'
+ENV CENTER='38.8995,-77.0269'
+ENV ZOOM='13'
+ENV LANGUAGE='en'
+ENV LABEL='Car (fastest)'
 
+# Create app directory
 RUN mkdir /src
 COPY . /src
 WORKDIR /src
 
+# Install app dependencies
 RUN apk add --no-cache sed nodejs && \
     cd /src && \
     npm install
```

Comment:
@freenerd This `COPY . /src` should declared after `npm install`.

Not an urgent, it's just faster to build the Docker container if you don't have to do `npm install` for every single minor change.